### ISSUE TITLE
Loading Images...

### ISSUE THREAD

**Issue by probityrules** [jiborobot/pixi-animate#30]
I was able to use the extension to successfully publish a shapes-only FLA (works great!), but when I published an FLA with bitmaps, I started getting the following error in pixi.js:

Texture.js:309 Uncaught Error: The frameId "Bitmap 4" does not exist in the texture cache

Do the published atlas png(s) need to be loaded apart from `PIXI.animate.load()`? There are no images listed in the published JS file's `assets` property, but I did notice that it includes an atlas JSON file. However, it doesn't appear to be automatically adding the image and its frames from that file to pixi's TextureCache (where the error above is trying to reference the unloaded asset).

**Comment by bigtimebuddy** [comment]
An atlas is automatically loaded when calling `PIXI.animate.load()`. 

Could you share an FLA that reproduces the issue? Are you about to preview in Animate, or same issue?

**Comment by probityrules** [comment]
Alright, I took about 3 steps back and it's working correctly when loading everything in the published html file. In our project we're using the latest version of both pixijs and pixi-animate, so I went ahead and swapped out the extension's published libraries, which were a little older, and the published html works fine with the new ones as well, so it seems to be something breaking the asset loading outside the scope of these two libraries. Some unrelated tangential code in our project must be interfering with things.

**Comment by bigtimebuddy** [comment]
I'm pushing a new release of the extension very soon, FYI.

**Comment by probityrules** [comment]
For what it's worth, I traced down the issue. I was trying to load multiple animations from different FLA's and their published namespaces were identical, causing things to overlap in illogical ways. For example: `var instance16 = new lib.Graphic15();` was being run in one JS export but pulling `Graphic15` from another JS export's definition for the class. Assets not having loaded yet was just a side-effect of this problem.

I guess I didn't fully understand that the namespace should be unique per FLA. Changed that now: problem solved!
